src/component/FolderToolBar/FolderButton/FolderButton.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import "./FolderButton.css";
+
+const FolderButton = ({ buttons, onFolderSelect }) => {
+  const [selectedButtonName, setSelectedButtonName] = useState("전체");
+
+  return (
+    <>
+      <button
+        className={selectedButtonName === null ? "selected" : ""}
```

Comment:
이렇게 간단하게 할 수 있겠네용 ㅎㅎ
```suggestion
        className={!selectedButtonName ? "selected" : ""}
```